public/assets/application-fbf25f87b816945058df04f4e29fe6317f0e0da6028c95c47944560f74bdf5a2.css

```diff
@@ -0,0 +1,299 @@
+/* line 1, /home/fabrizio/Documents/Sublime/Rails/railsmentor/app/assets/stylesheets/questions.scss */
```

Comment:
Also, making QPixel a paid mentor help app kinda breaks the spirit of QPixel IMO. It's God's gift for QPixel to be a FOSS clone of Stack Exchange, which is free for both the server and the client.